tasks/managed/extract-oot-kmods/extract-oot-kmods.yaml

```diff
@@ -0,0 +1,194 @@
+    - name: sourceDataArtifact
+      type: string
+      description: Location of trusted artifacts to be used to populate data directory
+      default: ""
+    - name: dataDir
+      description: The location where data will be stored
+      type: string
+      default: $(workspaces.data.path)
+  workspaces:
+    - name: input-data
+  results:
+    - name: sourceDataArtifact
+      type: string
+      description: Produced trusted data artifact
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  stepTemplate:
+    volumeMounts:
+      - mountPath: /var/workdir
+        name: workdir
+    env:
+      - name: IMAGE_EXPIRES_AFTER
+        value: $(params.ociArtifactExpiresAfter)
+      - name: "ORAS_OPTIONS"
+        value: "$(params.orasOptions)"
+      - name: "DEBUG"
+        value: "$(params.trustedArtifactsDebug)"
+  steps:
+    - name: skip-trusted-artifact-operations
+      computeResources:
+        limits:
+          memory: 32Mi
+        requests:
+          memory: 32Mi
+          cpu: 20m
+      ref:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: stepactions/skip-trusted-artifact-operations/skip-trusted-artifact-operations.yaml
+      params:
+        - name: ociStorage
+          value: $(params.ociStorage)
+        - name: workDir
+          value: $(params.dataDir)
+    - name: use-trusted-artifact
+      computeResources:
+        limits:
+          memory: 64Mi
+        requests:
+          memory: 64Mi
+          cpu: 30m
+      ref:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.taskGitUrl)
+          - name: revision
+            value: $(params.taskGitRevision)
+          - name: pathInRepo
+            value: stepactions/use-trusted-artifact/use-trusted-artifact.yaml
+      params:
+        - name: workDir
+          value: $(params.dataDir)
+        - name: sourceDataArtifact
+          value: $(params.sourceDataArtifact)
+    - name: extract-kmods
+      image: quay.io/konflux-ci/release-service-utils:0f82be4be43294b6a96846d87ef7f7c0b9e34267
+      computeResources:
+        limits:
+          memory: 512Mi
+        requests:
+          memory: 512Mi
+          cpu: 250m 
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+        export DOCKER_CONFIG
+        # Copy image to tmp
+        KMODS_PATH=$(params.kmodsPath)
+        SIGNED_KMODS_PATH=$(params.signedKmodsPath)
+        snapshot_json=$(get-resource "snapshot" "$(params.snapshot)")
+        sha_image=$(echo "$snapshot_json" | \
+                    jq -er '.metadata.annotations["build.appstudio.redhat.com/commit_sha"]')
+        source_image=$(echo "$snapshot_json" | \
+                    jq -r '.spec.components[].containerImage' | sed 's/@.*$//')
+        jq -r '.spec.components[].containerImage' <<< "$snapshot_json"
```

Comment:
sha_image and source_image can use `<<< "$snapshot_json"` just like this line does instead of echoing